src/Extensions/Logging/InitialisationLogger.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	/// <summary>
+	/// InitStaticLogger is the logger to be used before the proper ILogger from DI is set
+	/// not to be used as main logger
+	/// </summary>
+	public static class InitialisationLogger
+	{
+		/// <summary>
+		/// Instance of logger
+		/// </summary>
+		public static ILogger Instance { get; private set; } = LoggerFactory.Create(builder =>
+		 {
+			 builder.LoadInitialisationLogger();
+		 }).CreateLogger("Initial-Logging");
+
+		/// <summary>
+		/// Log on success
```

Comment:
I think we would need to clarify usage a bit, my idea was that this method would be used to indicate that service was successfully started (although I'm not sure its usefulness since generic host also does that, but that's something we used to do).